lib/grunt/utils.js

```diff
@@ -76,9 +76,9 @@ module.exports = {
 
 
   wrap: function(src, name) {
-    src.unshift('src/' + name + '.prefix');
-    src.push('src/' + name + '.suffix');
-    return src;
+    var files = ['src/' + name + '.prefix', 'src/' + name + '.suffix'];
+    files.splice(1, 0, src);
+    return files;
```

Comment:
This would be more readable:
```js
wrap: function(src, name) {
  return [`src/${name}.prefix`, ...src, `src/${name}.prefix`];
},
```
or even:
```js
wrap:(src, name) => [`src/${name}.prefix`, ...src, `src/${name}.prefix`],
```
Let's not be afraid of ES6+ in Node scripts. :)